doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -127,6 +127,10 @@ Matter
     In |NCS| Matter samples, the default reaction to migration failure is a factory reset of the device.
     To change the default reaction, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_FACTORY_RESET_ON_KEY_MIGRATION_FAILURE` Kconfig option to ``n``.
 
+  * Diagnostic logs provider that collects the diagnostic logs that come from the end user, network stack, or system crashes, and sends them to the Matter controller.
```

Comment:
To be honest I'm not sure if we need a docs changes at this stage, because this component is going to evolve and the docs will be likely changed as well. Nevertheless if we really want to do that, it would be good to mention, that network stack or end user logs are not yet supported.

Reply:
As long as the network stack and end user parts come before the next release, I'm fine with leaving the entry as is. If there's any chance that one or both might not make it, then I suggest changing the entry to:
```suggestion
  * Diagnostic logs provider that collects the diagnostic logs and sends them to the Matter controller.
    The current implementation collects logs from system crashes.
```
This entry can then be updated when the other parts are added, with the final expected addition changing "current implementation" to "the provider" to remove the implication that the functionality will be further expanded.